lib/improver/tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -739,6 +741,62 @@ def test_basic(self):
         self.assertIsInstance(nbcube, Cube)
         self.assertArrayAlmostEqual(nbcube.data, expected_data)
 
+    def test_complex(self):
+        """Test neighbourhooding with an array of complex wind directions"""
+        # set up cube with complex wind directions 30-60 degrees
+        a = 0.54066949+0.82535591j
+        b = 0.56100423+0.80502117j
+        c = 0.5+0.8660254j
+        d = 0.59150635+0.77451905j
```

Comment:
These numbers are fairly random.  I evaluated the sense of the answers looking at what came out of the complex_to_deg routine, which is a sensible set of outputs for an input array containing mostly 60s with a few 30s.  I could restrict the unit test to check only the outputs in degrees, but it would be cleaner to include checking the complex outputs since this locates the problem to either the neighbourhooding or WindDirection plugins.

Reply:
If the numbers represent a good test that you thought about then they are fine.